java/core/src/main/java/co/worklytics/psoxy/impl/RESTApiSanitizerImpl.java

```diff
@@ -474,17 +495,24 @@ Map<Endpoint, Pattern> getCompiledAllowedEndpoints() {
 
     @VisibleForTesting
     String effectiveRegex(Endpoint endpoint) {
+        //NOTE: java capturing groups names limited to A-Z, a-z and 0-9, and must start with a letter
```

Comment:
previously, we had no restriction on path parameter names, other than no `}`; so actually our github rules use snake_case, which is invalid

Openapi spec examples use camelCase
see : https://swagger.io/docs/specification/describing-parameters/#path-parameters

Reply:
i think restricting is better than trying to magically re-map `foo_bar` to `fooBar` or something, and hoping things don't collide.

while technically this is a breaking change, shouldn't affect many rule sets in the wild